workflow/rules/common.smk

```diff
@@ -2,8 +2,12 @@ import copy
 from datetime import datetime, timedelta
 import yaml
 import hashlib
+import logging
 import json
 
+LOG = logging.getLogger(__name__)
+logging.basicConfig(level=logging.WARNING)
```

Comment:
this reconfigures the root logger and overrides Snakemake’s console logger, is it intended? this results in all those `INFO:snakemake.logging:...` lines in the terminal output